src/payment/strategies/moneris/moneris-payment-initialize-options.ts

```diff
@@ -1,10 +1,17 @@
+import { HostedFormOptions } from '../../../hosted-form';
+
 import MonerisStylingProps from './moneris';
 
-export default interface MonerisaymentInitializeOptions {
+export default interface MonerisPaymentInitializeOptions {
     /**
      * The ID of a container where the Moneris iframe component should be mounted
      */
     containerId: string;
 
     style?: MonerisStylingProps;
+
+    /**
+     * Hosted Form Validation Options
+     */
+    form?: HostedFormOptions;
```

Comment:
Seems like the change you are doing here is to use the hosted form fields for vaulted instrument verification and the `HostedFormOptions` accepts parameters more than just instrument verification. For reference you can see this https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/hosted-form/hosted-form-options.ts#L20. As far as I can tell if the client passes those options it will not work. But here you are looking for https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/hosted-form/hosted-form-options.ts#L29 this particular interface to be accepted.

So maybe you need to describe a new interface to check for the supplied parameters?